### PR TITLE
Rough proof of concept of a scholarship page.

### PR DESCRIPTION
**Pull request by angaither**
- creates an 'about/scholarships' page
- currently themed as a table
- uses a 'feature flag' to allow for a deploy without exposing the page.
  -- to view locally in vagrant, run `drush vset dosomething_campaign_scholarship_page_is_active 1`

@mshmsh5000 @mikefantini

### CODE REVIEW

**Comment by mikefantini** [comment]
sweet! can we use high season end date as well as a "Deadline" field?

**Comment by angaither** [comment]
is the 'deadline field' a thing I don't know about?

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module [hunk 1]
```diff
@@ -41,8 +41,66 @@ function dosomething_campaign_menu() {
...
+    }
+    return t('No active scholarships right now, check back!');
+  }
+  return drupal_not_found();
+}
+
+/**
+ * Returns all published campaigns with a schalorship.
+ */
+function dosomething_campaign_get_schaolarships() {
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module]: Spelling.

**Comment by aaronschachter** [comment]
No deadline field that i know of either.  Seems like your'e referring to the end of a current run @mikefantini ? (which we dont have built yet)

**Comment by mikefantini** [comment]
sorry chatted with Andrea offline and forgot to document here cause I'm the worst. For now we're going to use end date of high season for deadline. @angaither @aaronschachter

**Comment by angaither** [comment]
Updated with high season "deadline"

**Comment by aaronschachter** [comment]
:+1:

**Pull request merged**